app/components/prosable-text.tsx

```diff
@@ -1,24 +1,20 @@
 import type { PortableTextTypeComponentProps } from "@portabletext/react";
 import { PortableText } from "@portabletext/react";
-import type { Language } from "prism-react-renderer";
-import Highlight, { defaultProps } from "prism-react-renderer";
-// TODO: Switch themes if we're not happy with GitHub
-import github from "prism-react-renderer/themes/github";
+import type { HighlightProps } from "prism-react-renderer";
+import { Highlight, themes } from "prism-react-renderer";
 
 import type { BlockContent } from "~/sanity/schema";
 import { urlFor } from "~/utils/imageBuilder";
 import { classNames } from "~/utils/misc";
 import { CapraImage } from "./capra-image";
 import { CapraLink } from "./capra-link";
 
-interface PrismProps {
-  code: string;
-  language: Language;
-}
+type Props = Pick<HighlightProps, "code" | "language">;
 
-function Prism({ code = "", language }: PrismProps) {
+function Prism({ code = "", language }: Props) {
```

Comment:
Jeg er uenig. Synes det egentlig er greit med konvensjonen for `NavnPåKomponentProps` for interfacer.
Når man har flere komponenter i en fil er det en nødvendighet, og jeg synes flere komponenter i en fil er helt greit, og ofte en god ide, og dermed følger det for meg at man kan holde seg til den konvensjonen.

Uansett, relativt en fillesak som ikke betyr så mye for meg

Reply:
Helt ok, jeg reverter den endringen. 😄